packages/react-component-library/src/components/List/List.tsx

```diff
@@ -34,7 +34,7 @@ export const List: React.FC<ListProps> = ({ children, ...rest }) => {
   )
 
   return (
-    <StyledList role="group" data-testid="list" {...rest}>
+    <StyledList data-testid="list" {...rest}>
```

Comment:
The error for this component was:

https://dequeuniversity.com/rules/axe/4.4/aria-required-parent

This was due to `listtem` roles not being within a `list` role. (On further investigation, the `group` role can be used to group multiple `listitem`s within a `list`, but isn't a replacement for a `list`.)

I've removed the roles completely as `ol` and `li` are already being used (and these have the correct roles automatically).